CODEOWNERS

```
@@ -0,0 +1,3 @@
+#ECCN:Open Source
+#GUSINFO:Heroku FE Dev Tools,Heroku CLI & Plugins
+* @heroku/frontend-dev-tooling
```

```suggestion
* @heroku/frontend-dev-tooling
```
I'm not sure what happened, but this should fix the error that's now showing up with this file. Hopefully.

---

Te error is because the team heroku/frontend-dev-tooling wasn't not a collaborator in this repo... Solved